store_test.go

```diff
@@ -68,7 +68,7 @@ func TestCookieDecode(t *testing.T) {
 	// Test with a nil hash key
```

Comment:
Need to split the test file out too.

Reply:
Technically it would pass as is, but I split and added a test to make sure the cookie attribute can be set/not set for current and legacy respectively